docs/dapps/nfts/intro-to-erc721s.md

```diff
@@ -0,0 +1,140 @@
+
+![Remix Deploy Page](intro-to-erc721s/7-deploy-page.png)
+
+Now, we need to change the environment that Remix will try to use to deploy the smart contract. Click on the `Environment` drop-down, and select `Injected web3`.
+
+![Remix Web3](intro-to-erc721s/8-web3.png)
+
+This should prompt you to connect with your MetaMask account. Once connected, you can verify the correct connection by checking that the Account number matches your MetaMask address.
+
+![Remix MetaMask](intro-to-erc721s/9-metamask.png)
+
+Now click on the `Contract` drop-down and select the contract you created and compiled. It should show up with the name you gave it in the Open Zeppelin Wizard.
+
+![Remix Contract](intro-to-erc721s/10-contract.png)
+
+Now, click deploy. This will open MetaMask and ask you to confirm the transaction. Click `Confirm`.
+
+![MetaMask Accept](intro-to-erc721s/11-accept.png)
+
+It may take a second, but once completed, your newly deployed contract will appear underneath the `Transactions Recorded` field.
+
+![Remix Record](intro-to-erc721s/12-transaction-record.png)
+
+Copy your contract’s address and open the [Snowtrace Testnet Explorer](https://testnet.snowtrace.io/). Paste your contract address in the search bar, and click `Search`.
+
+You will now see information on your contract. The first transaction you see should be the contract deployment you just did in the Remix IDE.
+
+![Snowtrace](intro-to-erc721s/13-snowtrace.png)
+
+## Minting an NFT
+
+Now that you have deployed the contract, you can mint the NFT. Go back to the Remix IDE tab and click on your contract to expand its information. A list of functions will appear that you can interact with.
+
+![Remix Functions](intro-to-erc721s/14-functions.png)
+
+The only function you are interested in is the `safeMint` function. Click the drop-down arrow for the function to expand the address field.
+
+![Remix safeMint](intro-to-erc721s/15-safemint.png)
+
+Now, copy your MetaMask address and paste it into this address field. This will send the NFT to your address when the mint function is called. After, hit `transact`.
+
+This will reopen MetaMask and ask you to verify the transaction. Click `Confirm` to mint your NFT.
+
+![Confirm Mint](intro-to-erc721s/16-confirm-mint.png)
+
+Once the transaction has been confirmed, you will see a green checkmark in the terminal at the bottom of the Remix IDE.
+
+![Remix Confirm Mint](intro-to-erc721s/17-remix-confirm.png)
+
+Head back to the Snowtrace Testnet explorer page for your contract and refresh it. You should now see a second transaction, your call to safeMint.
+
+![Snowtrace Mint](intro-to-erc721s/18-snowtrace-mint.png)
+
+By clicking on the Txn Hash, you see that your NFT was created!
+
+![Snowtrace Transaction](intro-to-erc721s/19-snowtrace-txn.png)
```

Comment:
I would suggest to add another section for Mainnet:
* Use the Mainnet setting for Core/Metamask
* You should have funds to cover costs during the operation, as there is no Faucet on Mainnet.